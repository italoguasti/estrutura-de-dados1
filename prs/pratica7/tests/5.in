500
Miguel 30 7 2024
Davi 31 3 2019
Isabella 28 3 1972
Pedro 7 5 1958
Gabriel 1 9 1969
Gabriel 30 7 1950
Alice 7 1 1969
Gabriel 14 1 1968
Pedro 6 4 1967
Isabella 2 8 1949
Alice 31 1 1961
Gabriel 16 12 1969
Pedro 10 12 2019
Gabriel 16 7 1978
Pedro 18 8 2014
Davi 13 8 1946
Miguel 1 10 1990
Davi 15 3 2001
Arthur 26 7 2020
Bernardo 16 3 1941
Arthur 27 8 2002
Gabriel 21 4 1995
Isabella 16 11 1995
Sophia 15 10 1937
Sophia 13 7 2003
Miguel 11 11 2001
Isabella 5 9 1930
Miguel 17 6 1975
Miguel 15 4 2024
Miguel 3 5 2010
Sophia 5 11 1941
Helena 4 9 1957
Isabella 18 5 2020
Arthur 25 5 1931
Bernardo 20 8 1967
Miguel 20 8 1942
Isabella 23 2 1959
Arthur 4 7 1985
Miguel 14 2 1982
Isabella 9 7 2024
Helena 11 1 2012
Bernardo 16 7 1992
Isabella 1 5 2013
Isabella 28 6 1959
Helena 24 10 1958
Bernardo 1 3 1930
Isabella 5 10 1933
Arthur 26 11 1948
Arthur 8 2 1938
Arthur 5 4 1931
Helena 21 4 1976
Davi 20 2 1957
Sophia 27 6 2010
Davi 13 5 2002
Isabella 12 10 1949
Sophia 23 11 1950
Bernardo 1 9 1963
Davi 12 10 1963
Pedro 2 8 2009
Miguel 22 2 2008
Helena 12 6 2016
Sophia 13 7 1979
Miguel 21 7 1978
Isabella 1 1 1967
Miguel 9 7 1976
Isabella 24 8 2011
Helena 14 1 2011
Pedro 1 7 2018
Davi 9 9 1970
Isabella 19 4 1997
Arthur 27 4 2024
Helena 5 7 1998
Davi 19 10 1962
Gabriel 1 6 1969
Helena 14 6 2024
Gabriel 9 8 2008
Sophia 9 1 2002
Isabella 22 3 1996
Bernardo 10 2 1990
Pedro 12 11 1935
Gabriel 28 9 1969
Gabriel 2 11 1935
Alice 5 2 1943
Pedro 24 9 1975
Alice 19 4 2001
Pedro 2 2 1997
Bernardo 29 10 1984
Davi 25 6 1973
Alice 16 1 2002
Alice 7 10 1998
Arthur 10 11 1950
Miguel 14 2 1992
Gabriel 5 10 1956
Isabella 1 10 2000
Davi 19 7 1987
Helena 28 3 1960
Sophia 4 2 1939
Sophia 7 12 1965
Sophia 13 9 1934
Arthur 7 9 1974
Arthur 19 7 1965
Gabriel 12 11 1986
Miguel 19 12 1956
Gabriel 1 1 1998
Helena 19 4 2014
Davi 31 10 1963
Gabriel 10 11 1947
Pedro 4 5 2018
Sophia 13 7 1993
Arthur 3 12 1973
Alice 30 4 1942
Helena 12 6 1950
Bernardo 14 9 2024
Isabella 18 5 2013
Sophia 22 2 2008
Helena 18 7 1970
Arthur 16 4 1992
Isabella 29 10 1967
Alice 12 8 1996
Alice 9 5 2003
Miguel 16 10 1955
Pedro 8 2 2010
Arthur 5 7 1980
Bernardo 30 8 1998
Pedro 30 11 1978
Gabriel 14 9 1994
Davi 2 6 1949
Arthur 28 12 1998
Bernardo 1 7 1957
Sophia 17 5 1961
Arthur 27 11 1951
Davi 29 11 1937
Miguel 17 11 1934
Davi 29 11 1960
Gabriel 25 2 1957
Sophia 30 6 1950
Davi 10 6 1995
Arthur 4 7 1966
Bernardo 22 6 1940
Davi 19 2 1967
Pedro 17 9 1949
Miguel 23 2 2018
Davi 8 6 1954
Gabriel 11 4 2008
Davi 25 2 1952
Gabriel 7 10 1972
Pedro 6 9 1983
Isabella 30 5 1989
Isabella 14 10 2012
Isabella 13 5 1976
Isabella 15 9 2015
Alice 14 11 1959
Sophia 27 4 2013
Miguel 12 4 1957
Miguel 6 1 1982
Davi 11 9 1967
Miguel 1 5 1954
Sophia 2 2 1950
Pedro 19 5 1960
Helena 1 4 1963
Sophia 21 5 1976
Pedro 26 11 2024
Bernardo 17 7 1947
Sophia 21 11 1966
Pedro 14 8 2005
Alice 11 5 1976
Sophia 13 6 1988
Miguel 2 7 2002
Helena 28 3 1934
Pedro 28 6 2023
Bernardo 30 3 1984
Davi 7 7 2000
Sophia 9 7 1994
Arthur 10 9 1992
Arthur 10 12 2005
Alice 22 8 1950
Bernardo 14 6 1930
Miguel 21 4 2015
Arthur 21 1 1949
Miguel 14 1 2015
Miguel 8 3 1980
Isabella 26 4 1989
Pedro 16 4 1937
Davi 2 8 1983
Alice 21 6 1970
Alice 13 1 1962
Pedro 19 5 1953
Arthur 17 4 1995
Davi 10 12 1953
Pedro 4 9 2012
Pedro 19 6 2007
Alice 5 11 1959
Sophia 23 4 2007
Arthur 6 4 2018
Alice 14 9 1983
Helena 28 5 1930
Arthur 25 2 1940
Davi 3 1 2003
Alice 14 9 1952
Sophia 30 8 1963
Davi 2 1 1981
Pedro 18 11 1930
Gabriel 19 1 1974
Miguel 24 7 1953
Arthur 19 4 1966
Pedro 11 6 1993
Davi 14 10 1981
Isabella 14 11 1995
Alice 21 7 1930
Helena 10 1 1994
Sophia 23 9 1962
Pedro 6 2 1953
Bernardo 28 9 1949
Isabella 1 12 1972
Alice 28 4 1964
Arthur 5 5 1940
Gabriel 3 12 2010
Alice 24 11 1984
Davi 13 10 1935
Pedro 27 5 1949
Sophia 24 4 2023
Sophia 11 5 1960
Sophia 17 9 1994
Helena 4 7 1994
Helena 4 6 1994
Pedro 23 12 1977
Helena 3 10 1939
Bernardo 9 10 1972
Arthur 20 2 1961
Davi 22 10 1982
Isabella 19 1 1949
Bernardo 9 3 2009
Gabriel 7 5 1949
Davi 3 5 1973
Pedro 14 9 1988
Arthur 5 6 1988
Pedro 24 6 2009
Pedro 5 6 1964
Helena 25 5 1957
Pedro 1 12 1968
Isabella 21 6 1981
Helena 3 7 1964
Sophia 4 1 2016
Davi 4 2 1931
Pedro 11 6 1971
Bernardo 15 12 2004
Helena 23 6 1988
Pedro 11 3 1968
Davi 9 3 2017
Helena 8 5 2005
Pedro 21 7 1981
Bernardo 28 1 1930
Isabella 14 2 1997
Alice 29 7 1944
Miguel 20 4 1970
Pedro 24 6 1996
Gabriel 1 9 1945
Bernardo 5 10 1969
Pedro 6 5 1982
Miguel 12 7 2014
Arthur 30 9 1946
Arthur 15 5 1977
Pedro 4 4 2024
Pedro 4 2 1946
Helena 20 2 2014
Gabriel 25 10 1983
Sophia 4 11 2007
Arthur 8 12 2003
Gabriel 26 3 1977
Arthur 11 4 1937
Alice 18 5 1972
Miguel 8 11 1991
Gabriel 22 9 1963
Arthur 31 3 1931
Miguel 22 3 2013
Isabella 16 7 1938
Davi 13 7 1988
Bernardo 27 6 1973
Pedro 18 3 2018
Bernardo 10 11 1982
Gabriel 12 2 1993
Arthur 26 3 1932
Alice 18 7 2019
Davi 25 12 1969
Isabella 14 3 1948
Miguel 3 9 2002
Alice 8 11 2020
Arthur 14 2 1984
Isabella 30 3 1953
Davi 8 10 2013
Davi 20 3 2015
Arthur 31 12 1990
Gabriel 23 8 1964
Alice 16 4 1970
Alice 29 10 1945
Isabella 16 5 2015
Pedro 8 9 1963
Sophia 21 5 1966
Miguel 7 12 2012
Helena 10 3 2014
Helena 11 7 1981
Isabella 26 9 1960
Pedro 8 9 1948
Davi 6 5 1944
Arthur 22 3 1975
Arthur 11 7 1941
Pedro 20 1 1991
Davi 12 4 2015
Isabella 24 5 1935
Isabella 17 2 1962
Helena 9 9 1986
Alice 9 8 1963
Sophia 22 8 1987
Alice 19 2 1981
Pedro 2 10 1984
Alice 26 12 1963
Bernardo 15 6 1989
Helena 7 9 1989
Helena 2 11 1930
Gabriel 17 11 1967
Gabriel 11 6 2002
Alice 8 2 1985
Arthur 12 7 2018
Gabriel 12 8 1992
Arthur 2 7 1979
Davi 1 10 1998
Isabella 5 1 1952
Bernardo 10 11 1961
Alice 11 10 1985
Miguel 28 2 2012
Miguel 12 8 1942
Gabriel 8 1 1962
Pedro 15 1 1988
Davi 26 9 1935
Davi 12 4 1950
Arthur 22 9 1979
Miguel 13 6 2019
Arthur 26 9 2003
Davi 7 11 2001
Miguel 27 9 1933
Gabriel 6 7 1976
Isabella 2 7 1937
Alice 31 8 1969
Bernardo 27 8 1957
Gabriel 29 9 1946
Pedro 11 5 1980
Isabella 26 6 1988
Bernardo 11 8 1937
Davi 13 12 2015
Isabella 15 7 2004
Alice 15 3 1938
Arthur 15 12 1938
Helena 15 1 1954
Gabriel 13 4 1973
Isabella 20 6 1995
Gabriel 4 2 1986
Pedro 2 12 1973
Bernardo 17 6 1976
Arthur 11 12 2002
Bernardo 11 10 2007
Arthur 21 11 2015
Sophia 19 12 1988
Pedro 22 5 1944
Miguel 12 12 1956
Helena 15 8 1942
Helena 23 7 1990
Gabriel 21 3 1989
Miguel 4 7 1932
Davi 3 3 1971
Gabriel 17 9 1974
Helena 12 7 1961
Davi 22 4 1940
Pedro 25 8 1947
Davi 6 3 1956
Alice 6 2 2000
Helena 13 11 1995
Pedro 18 2 1954
Arthur 30 4 1955
Helena 3 6 2024
Isabella 28 11 1933
Miguel 12 7 1980
Isabella 1 10 2002
Gabriel 20 6 1991
Bernardo 5 6 1944
Helena 25 5 1985
Miguel 22 6 1958
Sophia 9 11 1969
Arthur 21 1 1935
Bernardo 25 11 2004
Arthur 31 5 1964
Gabriel 30 6 2001
Miguel 23 1 1975
Sophia 30 5 1957
Helena 7 9 2023
Gabriel 21 9 1935
Gabriel 6 9 1973
Isabella 22 12 1990
Sophia 25 7 1954
Gabriel 5 8 1987
Davi 29 1 1977
Helena 21 6 1934
Alice 1 4 1993
Miguel 31 3 1934
Pedro 10 5 2014
Miguel 29 11 1945
Gabriel 25 7 2010
Bernardo 26 5 1988
Isabella 26 10 1962
Bernardo 3 6 1930
Gabriel 28 11 1979
Helena 1 11 1943
Alice 15 3 2020
Davi 9 2 1937
Isabella 24 1 2021
Bernardo 28 2 1974
Davi 6 12 1976
Davi 23 2 2023
Gabriel 14 9 1953
Isabella 5 12 1953
Bernardo 14 4 2010
Helena 26 8 1984
Bernardo 8 12 2003
Pedro 3 11 1969
Pedro 28 2 2023
Gabriel 20 12 2018
Alice 26 2 1936
Bernardo 3 3 1987
Isabella 3 7 2010
Miguel 14 8 1952
Gabriel 3 3 1981
Helena 20 12 1951
Helena 7 12 1947
Helena 3 3 1950
Miguel 26 11 2013
Alice 21 9 1948
Gabriel 14 3 1945
Alice 26 7 2019
Helena 26 3 1978
Isabella 19 9 2002
Miguel 1 12 2017
Miguel 2 8 1941
Pedro 13 12 1973
Isabella 16 3 1944
Alice 21 12 2007
Helena 26 8 2009
Helena 12 1 1935
Davi 5 1 1931
Pedro 26 11 1984
Miguel 3 4 1967
Isabella 17 12 1933
Isabella 14 1 1930
Helena 31 1 1989
Sophia 2 3 1944
Alice 28 7 1981
Helena 13 3 1988
Isabella 25 8 2015
Miguel 27 6 1941
Isabella 18 3 1962
Bernardo 9 2 1943
Miguel 20 12 2019
Bernardo 10 11 1939
Helena 31 1 1960
Alice 12 7 2020
Gabriel 8 12 1932
Sophia 22 5 1946
Gabriel 12 3 1978
Pedro 21 7 1953
Isabella 26 10 1995
Pedro 12 11 1978
Gabriel 22 3 1976
Pedro 31 5 1995
Helena 15 10 2009
Miguel 4 3 1989
Pedro 8 3 1956
Pedro 18 4 1984
Miguel 12 4 2021
Gabriel 17 10 1952
Gabriel 22 1 1958
Davi 21 3 1984
Sophia 23 4 1954
Pedro 13 12 1931
Pedro 23 1 2010
Miguel 30 12 1993
Pedro 29 11 1956
Isabella 27 4 2012
Isabella 20 2 2013
Davi 11 2 1953
Helena 27 4 1942
Pedro 7 12 1979
Isabella 21 12 2006
Alice 18 7 1953
Pedro 7 6 1947
Gabriel 17 5 1943
Sophia 24 3 1971
Davi 18 9 1985
Pedro 22 5 1950
Davi 14 7 2008
Gabriel 2 8 2000
Isabella 4 9 1970
Gabriel 12 12 1987
