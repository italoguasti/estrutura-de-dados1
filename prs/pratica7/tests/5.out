Analise.....: Comp | Movi
- Bubble....: 124750 | 64670
- Selection.: 124750 | 0493
- Insertion.: 65171 | 65171
